pages/Login.tsx

```diff
@@ -58,4 +128,8 @@ const Login = () => {
   )
 }
 
-export default Login
\ No newline at end of file
+export default Login
+
+function setTokenCookie(token: any) {
+  throw new Error('Function not implemented.');
+}
```

Comment:
This is not required

Reply:
forgot to remove these lines of code,
Done